roles/manage_cnpg/files/cnp-sandbox.yml

```diff
@@ -0,0 +1,10 @@
+# Example of a Basic PostgreSQL cluster
+apiVersion: postgresql.cnpg.io/v1
+kind: Cluster
+metadata:
+  name: cnp-sandbox
```

Comment:
Is this correct or should this also be `cnpg-sandbox`?

Reply:
this is correct

Reply:
updated

Reply:
I don't see this change